decentraland/unity-explorer
# fix: Make scenes UI appears behind Unity UI

## What does this PR change?
Fix #1353 

Make scenes UI appears always behind the Unity UI in order to avoid overlapping conflicts that could make important UIs (like the minimap) unusable.

![Screenshot_3](https://github.com/decentraland/unity-explorer/assets/64659061/0021bb6e-0cc6-485d-84ab-f89f22476c93)

## How to test it?
1. Enter the world.
2. Go to any scene that is creating any UI.
3. Check that the scene's UI is always behind the Unity's UI (minimap, chat, menus, etc.).
4. Also check that the name tags are not being overlaped by the scene's UI.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
